eclass/xorg-3.eclass

```diff
@@ -130,7 +130,7 @@ fi
 
 # Set up autotools shared dependencies
 # Remember that all versions here MUST be stable
-XORG_EAUTORECONF_ARCHES="x86-winnt"
+XORG_EAUTORECONF_ARCHES=""
```

Comment:
remove all the logic around this?

Reply:
I thought it could be useful to keep this for new arches like riscv and such, but on the other hand, I don't mind just doing it in ebuilds, and nowadays, we can ask Matt or Alan to cut a new release for us if we need it for any x.org software, so I'll drop it.